apps/src/constants.js

```diff
@@ -26,6 +26,7 @@ exports.TestResults = {
   NO_TESTS_RUN: -1,
 
   // The level was not solved.
+  GENERIC_FAIL: 0,                    // Used by DSL defined levels.
```

Comment:
Should we be concerned that there's a reason we weren't using 0 as a value previously (and had started using negative values)?